### PR TITLE
fixed startup and db handling

### LINKED ISSUE #24: process not starting
## Describe the bug

process is not starting

## Additional context

oneshot send is blocking and waiting for a subscription

### PR DESCRIPTION
**Pull request by aserowy**
close #24

### CODE REVIEW

File: src/persistence/mod.rs [hunk 1]
```diff
@@ -20,19 +20,33 @@ pub trait Repository: Send {
...
+        _ => {
+            debug!("Creating no persistence repository");
+            Box::new(NoPersistenceRepository {})
+        }
     }
 }
 
-pub async fn state_task(mut receiver: broadcast::Receiver<Event>, conf: &Configuration) {
-    let mut repository = create_repository(conf);
+pub async fn init_task(init_sender: oneshot::Sender<Vec<String>>, repo: &Box<dyn Repository>) {
```
Comment (thomaszub) [inline src/persistence/mod.rs:36]: See clippy

File: .devcontainer/docker2mqtt/config/configuration.yaml [hunk 2]
```diff
@@ -17,3 +15,5 @@ mqtt:
   port: 1883
   qos: 1 # default: 0
   # username: # default: None
+
```
Comment (thomaszub) [inline .devcontainer/docker2mqtt/config/configuration.yaml:18]: Why the change?

**Review by thomaszub — CHANGES_REQUESTED** [review]

File: src/persistence/mod.rs (hunk 1 above)
Comment (aserowy) [reply src/persistence/mod.rs:36]: will fix it and file a new commit

File: .devcontainer/docker2mqtt/config/configuration.yaml (hunk 2 above)
Comment (aserowy) [reply .devcontainer/docker2mqtt/config/configuration.yaml:18]: nvim.. u used to be cool! :D

Comment (thomaszub) [reply .devcontainer/docker2mqtt/config/configuration.yaml:18]: Never change a running config ;)

**Review by thomaszub — APPROVED** [review]

**Pull request merged**